component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -535,6 +535,70 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 					$replace[] = implode(", ", $catlinks);
 					$blank[] = "";
 					break;
+			case "{{CATEGORYLNK}}" || "{{CATEGORYLINK_RAW}}":
```

Comment:
Sorry I was supposed to remove the first block. I've done the same with an  || statement. I'll remove the first now.